packages/client/src/BrubeckNode.ts

```diff
@@ -213,4 +223,17 @@ export default class BrubeckNode implements Context {
             this.debug('closeProxyConnectionOnStream << %o', streamPartId, nodeId)
         }
     }
+
+    async getTrackerList(): Promise<SmartContractRecord[]> {
```

Comment:
This method is used only in` Broker.start()`. I'd remove the method and log just the smart contract address in `Broker.start()`. In Broker startup that gives us enough information. The list of trackers is not relevant to most of end-users, so there is maybe not too many other usages for this method.

(In case we want to keep the method, we could consider to query the data from `NetworkNode`. It is a long queue of methods (`TrackerManager.trackerRegistry.getAllTrackers()`) but that way we'd avoid storing the same information twice.)